util/CMakeLists.txt

```
@@ -245,8 +245,10 @@ if(ANDROID)
 endif()
 
 if(LINUX OR ANDROID)
+    find_package(CURL REQUIRED)
+    
     target_sources(crashpad_util PRIVATE
-        net/http_transport_socket.cc
+        net/http_transport_libcurl.cc
```

Then we can replace the hard-coded transport translation unit with a variable reference:
```suggestion
        ${HTTP_TRANSPORT_IMPL}
```